src/platform/packages/shared/response-ops/rule_form/src/rule_actions/rule_actions_connectors_body.tsx

```diff
@@ -126,13 +126,21 @@ export const RuleActionsConnectorsBody = ({
   const availableConnectors = useMemo(() => {
     return connectors.filter(({ actionTypeId }) => {
       const actionType = connectorTypes.find(({ id }) => id === actionTypeId);
-      const actionTypeModel = actionTypeRegistry.get(actionTypeId);
+      let actionTypeModel;
+      try {
+        actionTypeModel = actionTypeRegistry.get(actionTypeId);
+      } catch (error) {
+        // If the action type is not found, we do not show that connector
+        // eslint-disable-next-line no-console
+        console.warn(`Action type "${actionTypeId}" not found in action registry.`);
+        return false;
+      }
```

Comment:
```suggestion
      if (!actionTypeRegistry.has(actionTypeId)) {
         return false;
      }
      
       const actionTypeModel = actionTypeRegistry.get(actionTypeId);
       // rest of code
```

Also, I would suggest not console logging any warning in the logs if possible.